draft-ietf-quic-multipath.md

```diff
@@ -710,33 +706,25 @@ This also means that the same packet number can occur on each path and the
 packet number is not a unique identifier anymore. This requires changes to
 the ACK frame as well as packet protection as described in the following subsections.
 
-When multipath is negotiated,
-each path ID is linked to a separate packet number space.
+When multipath is negotiated, separate packet number space is linked to a Path ID.
 Each Path ID-specific packet number space starts at packet number 0. When following
 the packet number encoding algorithm described in {{Section A.2 of QUIC-TRANSPORT}},
 the largest packet number (largest_acked) that has been acknowledged by the
-peer in this new connection ID's packet number space is initially set to "None".
+peer in the Path ID-specfic packet number space is initially set to "None".
 
 ## Sending Acknowledgements
 
 The ACK_MP frame, as specified in {{ack-mp-frame}}, is used to
 acknowledge 1-RTT packets.
-Compared to the QUIC version 1 ACK frame, the ACK_MP frame additionally
-contains the receiver's Path ID associated with the Destination Connection ID
-to distinguish the path-specific packet number space.
-
-Acknowledgements of Initial and Handshake packets MUST be carried using
-ACK frames, as specified in {{QUIC-TRANSPORT}}. The ACK frames, as defined
-in {{QUIC-TRANSPORT}}, do not have a
-Path Identifier field to identify the path and this the packet number space.
-If the multipath extension has been successfully
-negotiated, ACK frames in 1-RTT packets acknowledge packets for the path with
-Path ID 0.
-
-As soon as the negotiation of multipath support is completed,
-endpoints SHOULD use ACK_MP frames instead of ACK frames to acknowledge application
-data packets, including 0-RTT packets, using the initial connection ID with
-Path ID 0 after the handshake concluded.
+Compared to the ACK frame as specified in {{Section 19.3 of QUIC-TRANSPORT}}, the ACK_MP frame additionally
+contains the receiver's Path ID to identify the path-specific packet number space.
+
+As multipath support is unknown during the handshake, acknowledgements of Initial and Handshake packets are sent using
+ACK frames.
+
+After the handshake concluded if negotiation of multipath support succeeded,
+endpoints SHOULD use ACK_MP frames instead of ACK frames to acknowledge
+1-RTT packets. Path ID 0 is used for the initial path.
```

Comment:
```suggestion
Compared to the ACK frame as specified in {{Section 19.3 of QUIC-TRANSPORT}}, the ACK_MP frame additionally
contains the receiver's Path ID to identify the path-specific packet number space.
As multipath support is unknown during the handshake, acknowledgements of Initial and Handshake packets are sent using ACK frames.
After the handshake concluded if negotiation of multipath support succeeded,
endpoints SHOULD use ACK_MP frames instead of ACK frames to acknowledge
1-RTT packets. Path ID 0 is used for the initial path.
```